libmscore/rendermidi.cpp

```diff
@@ -1164,6 +1164,11 @@ void Score::updateVelo()
                               continue;
                         const Dynamic* d = toDynamic(e);
                         int v            = d->velocity();
+
+                        // treat an invalid dynamic as no change, i.e. a dynamic set to 0
+                        if (v < 1)
+                              continue;
```

Comment:
Doesn't this make the "velocity change" property of dynamics be ignored?

Reply:
velocity change isn't relative to any previous velocity - it's a change relative to `v`. If v is invalid (i. e. less than 0) I think it's safe to ignore the dynamic completely, velocity change or not.

Reply:
I see now that this "velocity change" property was absent in 2.X so this patch wouldn't actually break things compared to that behavior. Still in new scores, couldn't it be useful to set the dynamics' velocity to 0 and velocity change to some larger value (say, +50 or -50) to make the dynamics just denote the fact that the following parts of a score should be played louder or quieter? This patch seems to block that option of using the dynamics. If we could instead just set `v` to the velocity that existed before this dynamics then it may both solve the issue and honor the velocity change.

Reply:
No, that's no what velocity change does. Velocity change is like with fortepianos and sforzandos, it's a loud bit followed immediately by a quiet bit without the use of a crescendo. Velocity change is not relative to any previous set velocity, as I've said.

Reply:
OK, then let's go this way. Thanks for the clarification!

Reply:
No problem, thanks for merging :)